src/averageAges.js

```diff
@@ -36,8 +43,19 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+function calculateWomenAverageAge(people, withChildren = false) {
+  const appropriateWoman = people.filter((person) => {
```

Comment:
'Woman' - singular form, 'women' - plural form
```suggestion
  const appropriateWomen = people.filter((person) => {
```